vale/Grafana/Shortcodes.yml

```diff
@@ -0,0 +1,18 @@
+extends: script
+message: |
+  Use `{{<` and `>}}` instead of `{{%%` and `%%}}`.
```

Comment:
Note the %% is intentional because this is a format string and %s is used to substitute in values. In order to have a literal % you have to escape it with %%.